cfme/tests/control/test_basic.py

```diff
@@ -526,6 +529,11 @@ def test_control_assign_actions_to_event(request, policy_class, policy, action):
         request.addfinalizer(policy.assign_events)
     else:
         prefix = policy.TREE_NODE if not policy.TREE_NODE == "Vm" else policy.TREE_NODE.upper()
+        if (
+            policy.TREE_NODE == "Physical Infrastructure" and
+            BZ(1700070, forced_streams=["5.10"]).blocks
```

Comment:
I don't think you want to use `forced_streams` here, this BZ is blocking for me without it, and it'll keep this workaround from auto-undoing itself when the bug is fixed.

```
In [2]: BZ(1700070).blocks
Out[2]: True
```

Reply:
Ah! Wasn't working this morning when I checked, I guess the BZ got triaged shortly thereafter, I'll add it in